### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by xylar**
Add xylar as a maintainer.

Drop python 2 checks, since it is not built.

Add pip check

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
The missing `geomet` dependency is giving me trouble downstream.

**Comment by xylar** [comment]
@anirrudh, @petercable and @scopatz, would you be willing to merge this and make me a maintainer?

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@conda-forge/core, I haven't received any feedback from maintainers in a month so I think this feedstock might be abandoned.  Would you be willing to merge this?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,25 +24,27 @@ requirements:
...
     - cython
     - libev  # [not win]
-    - msinttypes  # [win and py2k]
 
   run:
     - python
     - numpy
     - six >=1.9
-    - futures <=2.2.0  # [py2k]
-    - libev  # [not win]
```
Comment (wolfv) [inline recipe/meta.yaml:35]: libev is not needed?

**Comment by wolfv** [comment]
since this is passing `pip check` I am assuming the (new) dependencies are OK or even better than before. Merging!

Thanks for taking over @xylar

File: recipe/meta.yaml (hunk 1 above)
Comment (xylar) [reply recipe/meta.yaml:35]: @wolfv, thanks! I looked into this I'll add it back. It's optional but presumably desirable.

**Pull request merged**